service/src/test/java/bio/terra/catalog/common/StorageSystemTest.java

```diff
@@ -0,0 +1,12 @@
+package bio.terra.catalog.common;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class StorageSystemTest {
+  @Test
+  void fromStringIllegalArgumentException() {
```

Comment:
Could we add a positive test here as well? (Something `fromString` that takes a valid value)

Reply:
The reason there is none is because that case is covered in other tests. Should I still add it?

Reply:
If it is already covered, that's fine.